docs/installation/server-config/avro-schema.rst

```diff
@@ -1,4 +1,100 @@
 .. _install_ksql-avro-schema:
 
-Avro and Confluent Schema Registry
-==================================
\ No newline at end of file
+Avro and Schema Registry
+========================
+
+KSQL can read and write messages in Avro format by integrating with :ref:`Confluent Schema Registry <schemaregistry_intro>`.
+KSQL automatically retrieves (read) and registers (write) Avro schemas as needed and thus saves you from both having to
+manually define columns and data types in KSQL and from manual interaction with the Schema Registry.
+
+.. contents:: Contents
+    :local:
+
+Supported functionality
+^^^^^^^^^^^^^^^^^^^^^^^
+
+KSQL currently supports Avro data in the Kafka message values.
+
+The following functionality is not supported yet:
+
+-  Message keys in Avro format are not supported. Message keys in KSQL are always interpreted as STRING format, which means
+   KSQL will ignore Avro schemas that have been registered for message keys.
+-  Avro schemas with nested fields are not supported yet. This is because KSQL does not yet support nested columns.
+
+Configuring KSQL for Avro
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You must configure the API endpoint of |sr| by setting ``ksql.schema.registry.url`` (default: ``http://localhost:8081``)
+in the KSQL configuration file (``<path-to-confluent>/etc/ksql/ksql-server.properties``) that you use to start the KSQL
```

Comment:
Nit: KSQL *server* configuration file.  We can then remove "that you use to start the KSQL Server".